src/testcafe/testcafe.builder.ts

```diff
@@ -102,7 +102,8 @@ async function execute (
         return { success: true };
 
     }
-    catch (e) {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    catch (e: any) {
```

Comment:
Please try to do it without the `any` type. You should add conditions for `e` in the way described [here](https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#cfa-aliased-conditions):
```suggestion
    catch (e) {
        const message = e instanceof Error ? e.message : 'Testcafe run failed!';
```

Reply:
Thank you, I eliminated the explicit any by applying your suggestion.